src/som/vmobjects/SClass.java

```diff
@@ -66,6 +72,8 @@
 
   @CompilationFinal private ClassFactory instanceClassGroup; // the factory for this object
 
+  @CompilationFinal protected AbstractSerializationNode serializer;
```

Comment:
This is a node that specializes itself? I am not sure this is going to work if any `rewrite` is going to be used.
But I only saw specializations, so, might be ok.